src/test/java/PlaylistTests.java

```diff
@@ -1,2 +1,196 @@
-public class PlaylistTests {
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlaylistTests extends BaseTest{
+
+    @Test(enabled = true)
+    public void addSongTest() throws InterruptedException {
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/214276896-c1a2131f-203d-4cb0-ae1a-4e08ac8c6a30.png)